tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "outDir": "./dist",
+    "allowJs": true,
+    "target": "es6"
```

Comment:
under this I'd highly recommend noImplicitAny true and strictNullChecks true. It's worth it to do it I promise